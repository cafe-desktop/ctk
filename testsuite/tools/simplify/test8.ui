<!-- non-default child property remains -->
<interface>
  <object class="CtkBox">
    <child>
      <object class="CtkLabel">
      </object>
      <packing>
        <property name="expand">True</property>
      </packing>
    </child>
  </object>
</interface>
