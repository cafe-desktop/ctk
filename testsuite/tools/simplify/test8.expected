<!-- non-default child property remains -->
<interface>
  <object class="GtkBox">
    <child>
      <object class="GtkLabel"/>
      <packing>
        <property name="expand">1</property>
      </packing>
    </child>
  </object>
</interface>
